apps/ai-pipeline/app/pipeline/enrichment.py

```
@@ -0,0 +1,57 @@
+                response_format={"type": "json_object"},
+                timeout=settings.llm_timeout,
+                api_key=settings.gemini_api_key,
+            )
+
+            content = response.choices[0].message.content
+            data = json.loads(content)
+            result = EducationalContent.model_validate(data)
+            logger.info("교육 콘텐츠 생성 성공: %s", result.title)
+            return result
+
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)